src/app/component/NewsletterSubscription/images/letter-icon.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="14px" viewBox="0 0 18 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>3946694A-9211-4FC1-AAA0-52ED9CD30EF4</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="budo&amp;fitness---footer-mobile" transform="translate(-271.000000, -610.000000)" fill="#FFFFFF" fill-rule="nonzero">
```

Comment:
I do not think budo & fitness icon is alright here.

Reply:
This SVG is not optimized.

Reply:
I would appreciate if you would make this icon using CSS. Like we did other icons in `_icons.scss`.

Reply:
This icon is pretty simple to be done in CSS

Reply:
Done